crates/holochain/src/conductor/tests/install_app_bundle.rs

```diff
@@ -513,256 +440,179 @@ async fn cells_by_dna_lineage() {
     pretty_assertions::assert_eq!(linx, btreeset![app_cells(&app1, &[1]),]);
 }
 
-/// Test all possible combinations of Locations and network seeds:
 #[tokio::test(flavor = "multi_thread")]
-#[cfg(feature = "glacial_tests")]
-#[ignore = "this is a really useful comprehensive test, but it's so dang slow"]
-async fn network_seed_affects_dna_hash_when_app_bundle_is_installed() {
+async fn use_existing_happy_path() {
```

Comment:
The diff in this file is messy because I moved a bunch of stuff to another file. All of the deletions around this point are just code moved out of the file. Just read the additions from here on out, don't worry about the changes.